pkg/api/admin/config.go

```diff
@@ -8,6 +8,9 @@ import (
 
 // Config holds the cluster admin config structure
 type Config struct {
+	// SecurityPatchPackages defines a list of rpm packages that fix security issues
+	SecurityPatchPackages *[]string `json:"securityPatchPackages,omitempty"`
```

Comment:
I'd prefer CVEPackages, they can be installed with `yum update --cve` and etc. so the name kinda has existing history.